README.md

```diff
@@ -79,6 +79,10 @@ in the Kubernetes documentation.
 
 Please submit an issue at: [Issues](https://github.com/ceph/ceph-csi/issues)
 
-## Slack Channels
+## Contact
 
-Join us at [Rook ceph-csi Channel](https://rook-io.slack.com/messages/CG3HUV94J/details/)
+Please use the following to reach members of the community:
+
+- Slack: Join our [slack channel](https://cephcsi.slack.com)
+- Forums: [rook-dev](https://groups.google.com/forum/#!forum/ceph-csi)
```

Comment:
Header text reads "rook-dev" and should change to cephcsi-dev?

If this is a dev channel should the google groups be renamed with the dev suffix?

Reply:
Changed, forgot to push, but its now.

Reply:
Is the channel a dev channel? If so can we suffix it with "-dev"?

Reply:
not really shyam, its meant for all ceph-csi communication. @ShyamsundarR